src/app/server/views/404.hbs

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE HTML>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Page Not Found</title>
+        <link rel="stylesheet" href="../compiled/basicApp.css">
```

Comment:
This feels like it is pointing in the wrong place. Should we use some generic css  + some app specific css if needed?

Reply:
Yeah, that's a temporary hack. Should have a build step for generic css to be available to templates but also potentially the user provided index.html. 

More generally, I guess that index.html should be able to include its own css, image files, other pages etc - so  maybe there should be a folder path for the user to provide all that if they want to. The index.html itself could go in there too.

Reply:
Actually, I think we can just say that if the index.html files needs additional files, they can just go in the /files path, and it's up to the instance to organise them how they'd like. I'll add an example of that.